snaps_tests/demos_tests/test_downloader_with_wiki_parts.py

```diff
@@ -27,7 +27,8 @@ def test_downloader_with_wiki_parts(self):
         snap_path = self.build_snap(self.snap_content_dir)
         self.install_snap(snap_path, 'downloader', '1.0')
         expected = (
-            '.*<title>The leading OS for PC, tablet, phone and cloud '
+            '.*<title>The leading operating system for PCs, tablets, '
```

Comment:
This should be reverted as it was fixed in master with a more permanent solution.